modules/kubernetes-addons/aws-node-termination-handler/locals.tf

```diff
@@ -1,6 +1,6 @@
 locals {
   namespace            = "kube-system"
-  name                 = "aws-node-termination-handler"
```

Comment:
these will fail since the chart name is something we don't control but set by the chart. the release name can change, the chart name cannot

Reply:
Changed naming here and verified it was all created and working.